src/category_theory/monoidal/category.lean

```diff
@@ -0,0 +1,384 @@
+-- Copyright (c) 2018 Michael Jendrusch. All rights reserved.
+-- Released under Apache 2.0 license as described in the file LICENSE.
+-- Authors: Michael Jendrusch, Scott Morrison
+import category_theory.monoidal.tensor_product
+import category_theory.natural_isomorphism
+import tactic.basic
+import tactic.slice
+
+open category_theory
+open tactic
```

Comment:
Do you need to `open tactic`? A quick look through this file didn't show me why you should open it.

Reply:
Fixed.